FirebasePerformance/Sources/FPRNanoPbUtils.h

```diff
@@ -0,0 +1,191 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <TargetConditionals.h>
+#if __has_include("CoreTelephony/CTTelephonyNetworkInfo.h") && !TARGET_OS_MACCATALYST
```

Comment:
Please test locally with Xcode 13 beta if not already done so - since the iOS/Catalyst define hierarchy has changed.

Reply:
- Cocoapods: all tests are passing.
- SPM: The Fireperf library could not be built because we marked `NS_EXTENSION_UNAVAILABLE("FirebasePerformance does not support app extensions at this time.")` for `FIRTrace`.  I will create another PR to address this very soon.